gabc/index.html

```diff
@@ -374,23 +344,27 @@ <h2 id="text">Text</h2>
+<p>The initial is automatically recognized by gregorio as the first letter.  You can use the gabc attribute <code>initial-style</code> in the header to set its size.  The values corespond to number of lines the intial takes up: <code>0</code> (there is not initial), <code>1</code> (the default), or <code>2</code> (the initial takes up two lines).</p>
 
-<p>Note that in the case of a syllable composed of one letter only, gregorio does not automatically add an hyphen under the first notes. This is due to the fact that it's easy to add it in gabc, and it would be difficult to remove it if it was automatically added.</p>
+<p>Note that in the case of a syllable composed of one letter only, Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> automatically adds a hyphen under the first notes.  To disable this feature, use the command <code>\gresetemptyfirstsyllablehyphen{auto}</code>.</p>
 
 <h3 id="specialcharacters">Special characters</h3>
 
-<p>It can happen that the texts of Gregorian chants contain non-unicode characters, so we must consider how to notate them.   Special characters used in Gregorian notation are <code>ý</code>, <code>ǽ</code>, <code>œ́</code>, the crossed R (for <em>Response</em>), the crossed V (for <em>Versicle</em>) and the crossed A (for <em>Antiphon</em>). Of these characters, only the crossed A is not to be found in the Unicode table.</p>
+<p>It can happen that the texts of Gregorian chants contain unicode characters which are not easily accesible from the keyboard.   Examples of this sort of characters used in Gregorian notation are <code>ý</code>, <code>ǽ</code>, <code>œ́</code>, the crossed R (℟ for <em>Response</em>), and the crossed V (℣ for <em>Versicle</em>).  There is also the possibility of a non-unicode character such as the crossed A (for <em>Antiphon</em>).</p>
 
-<p>In the case where it is not possible to insert special characters using the keyboard, this can be done using <code>sp</code> markup tags, for example <code>&lt;sp&gt;R/&lt;/sp&gt;</code> for the crossed R, <code>&lt;sp&gt;'ae&lt;/sp&gt;</code> for accented <em>æ</em>.</p>
+<p>Gregorio will allow the use of any unicode character directly entered via your platform's character picker or a keyboard shortcut.  In the case where it is not possible to insert special characters in one of these fashions, you can be use <code>&lt;sp&gt;</code> markup tags, for example <code>&lt;sp&gt;R/&lt;/sp&gt;</code> for the crossed R, <code>&lt;sp&gt;'ae&lt;/sp&gt;</code> for <em>ǽ</em>.  This also works for the crossed A, even though it is not in the unicode table: <code>&lt;sp&gt;A/&lt;/sp&gt;</code></p>  Finally, you can make use of the <code>&lt;v&gt;</code> markup tags to enter a character using it's <span class="tex">T<span class="epsilon">e</span>X</span> code: <code>&lt;v&gt;{\ae}&lt;/v&gt;</code> for <em>æ</em>.
```

Comment:
`in one of these fashions` should be `in either of these ways`.  Also the same thing about `barred` vs. `crossed`.